node/metrics/createQuote.ts

```diff
@@ -7,18 +7,6 @@ type UserData = {
   roleId: string
 }
 
-type SessionData = {
```

Comment:
Why was this type removed?

Reply:
It was just moved to node/typings.d.ts to be used anywhere.